### ISSUE TITLE
pbench-iozone is missing options used in perf testing

### ISSUE THREAD

**Issue by wilkins11** [distributed-system-analysis/pbench#353]
I routinely use the following options when characterizing sequential I/O in out test envs.

      -c    Include close in the timing calculations
      -C    Show bytes transferred by each child in throughput testing
      -e    Include flush (fsync,fflush) in the timing calculations
      -w    Do not unlink temporary file
     -+n    No retests selected

Is it possible to have these options added to pbench-iozone?

**Comment by wilkins11** [comment]
Plz back-burner this request while we see if pbench-fio will suit for sequential testing.

**Comment by bengland2** [comment]
@wilkins11  +1, I think pbench-fio can match pbench-iozone pretty closely with 

    fio --exitall=1 --create_on_open=1 --create_serialize=0 --create_on_open=1 ...

The one fly in the ointment - This means that for read tests you have to re-run the sequential write test without --exitall to ensure that all files have been written completely, then drop cache, then do the read test.

**Comment by portante** [comment]
@atheurer, @ndokos, what do you think about adding a "how to do iozone using fio" documentation page, and dropping `pbench-iozone` entirely?

**Comment by bengland2** [comment]
I agree with dropping iozone.  I actually have seen fio match iozone results exactly for raw device testing using --exitall=1.  

As Tim stated above, there are some subtleties to fio parameters that people need to be aware of when doing filesystem testing, and pbench-fio is hiding all this from them.  Either pbench-fio has to understand how to make use of fio parameters and do it automatically on behalf of the user, or pbench-fio has to let the user guide it.
